quaterion/train/xbm/xbm_buffer.py

```diff
@@ -0,0 +1,62 @@
+from typing import Tuple
+
+import torch
+from torch import LongTensor, Tensor
+
+from quaterion.train.xbm.xbm_config import XbmConfig, XbmDevice
+
+
+class XbmBuffer:
+    """A buffer implementation to hold recent N embeddings and target values.
+
+    Inspired by https://github.com/msight-tech/research-xbm/blob/master/ret_benchmark/modeling/xbm.py
+
+    Args:
+        config: Config class to configure XBM settings.
```

Comment:
please, add `embedding_size` to args, otherwise in docs it seems like there is only one param to manage